docs/documentation/server_admin/topics/sso-protocols/con-oidc-auth-flows.adoc

```diff
@@ -361,9 +366,20 @@ on the clients.
 If the client is not enabled with front-channel logout, then {project_name} is going to try first to send logout requests through the back-channel
 using the <<_back-channel-logout-url, Back-Channel Logout URL>>. If not defined, the server is going to fall back to using the <<_admin-url, Admin URL>>.
 
+If even the Admin URL is not specified, {project_name} will not propagate logout to the client. This can be still sufficient for many client application deployments. For
+instance when the client application is frontend javascript application, it may rely on the <<_oidc-logout-session-management, Session Management>> and hence there is no need to send
```

Comment:
```suggestion
instance, when the client application is a frontend javascript application, it may rely on the <<_oidc-logout-session-management, Session Management>> and hence no need exists to send
```